circle/functions

```diff
@@ -336,6 +361,17 @@ docker_build_and_gcloud_push() {
   gcloud_docker_push ${1} ${2}
 }
 
+ibm_login() {
+  install_ibm_cloud_cli || return 1
+
+  info "Install the Container Registry plug-in...."
+  bx plugin install container-registry -r Bluemix
```

Comment:
IMO the plugin installation should be performed in the the `install_ibm_cloud_cli` function. The `ibm_login` should only be concerned with the user authentication